openmc/data/photon.py

```diff
@@ -379,6 +391,41 @@ def to_hdf5(self, group, shell):
                  _SUBSHELLS, range(len(_SUBSHELLS)))
             group.create_dataset('transitions', data=df.values.astype(float))
 
+    def energy_fluorescence(self, shell):
+        """Compute expected energy of fluorescent photons for the shell
+        """
```

Comment:
Need a "Parameters" section describing the arguments and a "Returns" section describing the return value.

Reply:
done